token/services/identity/config/identity.go

```diff
@@ -16,25 +17,12 @@ type Config interface {
 	UnmarshalKey(key string, rawVal interface{}) error
 }
 
-type Identity struct {
-	ID        string      `yaml:"id"`
-	Default   bool        `yaml:"default,omitempty"`
-	Path      string      `yaml:"path"`
-	CacheSize int         `yaml:"cacheSize"`
-	Type      string      `yaml:"type,omitempty"`
-	Opts      interface{} `yaml:"opts,omitempty"`
-}
-
-func (i *Identity) String() string {
-	return i.ID
-}
-
 type Wallets struct {
-	DefaultCacheSize int         `yaml:"defaultCacheSize,omitempty"`
-	Certifiers       []*Identity `yaml:"certifiers,omitempty"`
-	Owners           []*Identity `yaml:"owners,omitempty"`
-	Issuers          []*Identity `yaml:"issuers,omitempty"`
-	Auditors         []*Identity `yaml:"auditors,omitempty"`
+	DefaultCacheSize int                          `yaml:"defaultCacheSize,omitempty"`
+	Certifiers       []*driver.ConfiguredIdentity `yaml:"certifiers,omitempty"`
```

Comment:
We could also use `map[RoleType][]*ConfigurationIdentity`

Reply:
this struct is unmarshalled from yaml